### DISCUSSION TITLE
Combine Search with the QExpansion

### DISCUSSION THREAD

**Discussion by Code-Case** [quasarframework/quasar#9060] [General - Components / Directives / etc] [answered]
Hi,

is it possible to get the Search with the QExpansion combined in Quasar?
I would like to use the QExpension-item and a search on top together to filter the List, nearly the same like: 

https://codepen.io/metalsadman/pen/eYgbMVV

But with the https://quasar.dev/vue-components/expansion-item#example--playing-with-inset-levels

Thank you for quasar, its awesome!

**Comment by ontwikkelfabriek** [comment, marked as answer]
I don't think the QExpansionItem is very suitable for filtering/searching. Instead of using the QExpItem you could maybe take a look at QTree.

Here's a codepen example to filter a Qtree with a search input:
[Qtree with a search input](https://codepen.io/xariusdesai/pen/RwogmdE)

**Reply by Code-Case** [reply]
Hi and thank you, thats a very nice component for this. Did you know if its possible to show the icons/buttons on the right side and not on the left? Maybe with a slot or some other tweaking? Looks like all QTree examples have their icons/buttons on the left side.

**Reply by ontwikkelfabriek** [reply]
> Did you know if its possible to show the icons/buttons on the right side and not on the left? Maybe with a slot or some other tweakin

Yes you can use the `default-header` slot:
https://codepen.io/ontwikkelfabriek/pen/MWJRKxX

**Reply by Code-Case** [reply]
Hi and thanks, I mean the whole tree navigation wich is now on the left side of the tree, sorry :)

**Reply by ontwikkelfabriek** [reply]
Here you go:
https://codepen.io/ontwikkelfabriek/pen/MWJRKxX

Sorry I cheated with `class="flip-horizontal"`. I have to pass on this one...

**Reply by Code-Case** [reply]
Thanks, learning every day what Quasar can do :)
